CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 ## [Unreleased]
 
+# 1.12.1
+### Changed
+- Debify now receives the flag `--output` as input to indicate the file type that it should package (e.g `rpm`). If this 
+  flag is not given, the default value is `deb`.
+  <link to github issue>
+### Fixed
+
+* Same Configuration to deb and rpm bug that happens in Conjur Evoke CICD. 
+    [conjurinc/debify#56](https://github.com/conjurinc/debify/issues/56)
```

Comment:
you can write this:
```
- Debify now receives the flag `--output` as input to indicate the file type that it should package (e.g `rpm`). If this 
  flag is not given, the default value is `deb`.
  [conjurinc/debify#56](https://github.com/conjurinc/debify/issues/56)
```

we don't need the `Fixed`.

Reply:
Done